docs/partners/1-badging/registry-of-ed-fi-badges.mdx

```diff
@@ -0,0 +1,535 @@
+---
+sidebar_position: 3
+---
```

Comment:
For now, I recommend adding `hide_table_of_contents: true` to the front matter, so that the table can take up more horizontal space on the page. FYI, [here is the list](https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs#markdown-front-matter) of all available frontmatter properties. It is annoyingly hard to find from the main Docusaurus page.